tests/specs/lemmas.k

```diff
@@ -267,6 +267,12 @@ module LEMMAS-JAVA [symbolic, kast]
     rule #buf( 32, DATA ) ==K BA:ByteArray     => DATA ==Int #asInteger(BA)
       requires #sizeByteArray(BA) <=Int 32                                      andBool  #isConcrete(BA) [simplification]
 
+    //For #bufStrict simplification in benchmarks
+    rule 0 <Int 2 ^Int I           => true
+      requires 0 <=Int I                                                        [simplification]
+    rule 0 <=Int I *Int 8          => 0 <=Int I                                 [simplification]
+    rule 0 <=Int #ceil32(I) -Int I => true                                      [simplification]
```

Comment:
Is this true if `I <Int 0`? I don't think so. Maybe you need a side-condition that `0 <=Int I`?

Reply:
thanks, fixed